### ISSUE TITLE
Color Contrast for Critical and Online Colors - Light Theme

### ISSUE THREAD

**Issue by MaximKipot** [grafana/grafana#22803]
Good day,

unfortunately, the colors used for alerts within notification channels are not accessible, meaning the contrast to the background is not enough.

**Steps to reproduce**:
open a graph and create a notification channel
in the alert tab define the condition and a value above which the value is alerting
in the state history you will see all the states being green, red or orange depending on the definition above

**Actual Result**:
<img width="289" alt="Screenshot 2020-03-16 at 12 11 00" src="https://user-images.githubusercontent.com/1902515/76750870-396a5980-677f-11ea-8f6c-d1ee9cf071a1.png">

**Expected Result**
<img width="323" alt="Screenshot 2020-03-16 at 12 09 28" src="https://user-images.githubusercontent.com/1902515/76750891-3cfde080-677f-11ea-8ea5-7e80f5d5404d.png">

So the values found within _variables.light.generated.scss:
$online: #369b4f;
$warn: #f79520;

should be changed into smth like:
$online: #2b7b3f;
$warn: #a25b06;

**Relevant WCAG Criteria:** [1.4.3 WCAG Criterion](https://www.w3.org/TR/2008/REC-WCAG20-20081211/#visual-audio-contrast-contrast)

**Environment**:
- Grafana version: 6.7.0-pre
- Data source type & version: TestData DB coming with Simple dummy streaming example
- User OS & Browser: MacOS Mojave, Mozilla and Chrome

**Comment by dprokop** [comment]
@jessover9000 do you have any suggestions about colors to use to improve the accessibility in this case?

**Comment by dprokop** [comment]
as of https://github.com/grafana/grafana/pull/22831#issuecomment-599980617 : We are not ready to update our color palette yet, it will require a bigger overhaul and testing or a separate theme.

In the meantime upvote: #18041
